devex/cmd/onclustertesting/opts.go

```
@@ -18,9 +18,8 @@ type opts struct {
 	containerfilePath        string
 	containerfileContents    string
 	poolName                 string
-	injectYumRepos           bool
-	waitForBuildInfo         bool
-	enableFeatureGate        bool
+	injectYumRepos  bool
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)